### PR TITLE
Update IR documentations

### PR DESCRIPTION
**Pull request by IronPan**
**Description of your changes:**
Update IR documentations

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: api/v2alpha1/pipeline_spec.proto [hunk 1]
```diff
@@ -669,15 +620,13 @@ message PipelineDeploymentConfig {
     // The query to fetch artifacts.
     message ArtifactQuerySpec {
       // The filter of the artifact query. The supported syntax are:
-      // - `contexts.name='<context name>'`
+      // - `in_context("<context name>")`
       // - `artifact_type='<artifact type name>'`
       // - `uri='<uri>'`
       // - `state=<state>`
-      // - `properties['key']='value'`
```
Comment (dushyanthsc) [inline api/v2alpha1/pipeline_spec.proto:676]: Wouldn't the equivalent metadata fields be supported.

So the existing ones `properties['key']='value'` will be `metadata.key.<type-name> = value` and `custom_properties['key']='value'` will be `metadata.key.<type-name> = value`

**Review by dushyanthsc — APPROVED** [review]

File: api/v2alpha1/pipeline_spec.proto (hunk 1 above)
Comment (IronPan) [reply api/v2alpha1/pipeline_spec.proto:676]: ack.

**Comment by chensun** [comment]
/lgtm

@jiyongjung0 to take a look as well.

**Comment by jiyongjung0** [comment]
LGTM from TFX side. It seems like TFX has already migrated and not using deprecated fields any more.

**Comment by mkbhanda** [comment]
@jiyongjung0 or @chensun -- one of you needs to /approve

**Pull request merged**